yadm

```diff
@@ -1552,6 +1531,20 @@ function process_global_args() {
 
 }
 
+function qualify_path() {
```

Comment:
Is it possible to use `relative_path()` ?

Reply:
To be more clear, I mean the existing `relative_path()` function (not renaming this).

Reply:
Not that I can think of. Doesn't `relative_path` do the opposite of `qualify_path`?

Reply:
Yes, of course, that's right...